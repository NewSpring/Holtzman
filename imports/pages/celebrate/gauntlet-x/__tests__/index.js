import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";
import GauntletX from "../";

const generateComponent = () => (
  <GauntletX />
);

it("renders", () => {
  const wrapper = shallow(generateComponent());
  expect(shallowToJson(wrapper)).toMatchSnapshot();
});
