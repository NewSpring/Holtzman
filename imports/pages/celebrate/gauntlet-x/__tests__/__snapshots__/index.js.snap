exports[`test renders 1`] = `
<div
  className="display-inline-block text-center one-whole ">
  <div
    className="background--primary soft-double-top@lap-and-up"
    style={
      Object {
        "overflow": "visible",
      }
    }>
    <Stats>
      <Connect(FitTextWithoutData)
        compressor={1.5}>
        <h3
          className="flush-bottom soft-half-bottom soft-double-top text-tertiary italic"
          style={
            Object {
              "fontFamily": "ff-meta-serif-web-pro, serif",
            }
          }>
          At Gauntlet X we saw
        </h3>
      </Connect(FitTextWithoutData)>
      <Connect(FitTextWithoutData)
        compressor={0.3}>
        <h1
          className="uppercase text-light-primary"
          style={
            Object {
              "fontWeight": "900",
            }
          }>
          1,066
        </h1>
      </Connect(FitTextWithoutData)>
      <Connect(FitTextWithoutData)
        compressor={0.65}>
        <h1
          className="uppercase push-back-top text-light-primary"
          style={
            Object {
              "fontWeight": "900",
            }
          }>
          Salvations
        </h1>
      </Connect(FitTextWithoutData)>
      <Connect(FitTextWithoutData)
        compressor={1.7}>
        <h3
          className="flush-bottom push-back-half-top soft-bottom text-tertiary italic"
          style={
            Object {
              "fontFamily": "ff-meta-serif-web-pro, serif",
              "fontSize": "23px",
            }
          }>
          because you cared about
        </h3>
      </Connect(FitTextWithoutData)>
      <Connect(FitTextWithoutData)
        compressor={0.55}>
        <h1
          className="uppercase text-light-primary"
          style={
            Object {
              "fontSize": "53px",
              "fontWeight": "900",
            }
          }>
          Students
        </h1>
      </Connect(FitTextWithoutData)>
    </Stats>
    <div
      className="soft-double-ends" />
    <Leaves>
      <SolidLeaf
        className="locked-bottom locked-right"
        color="text-secondary"
        size="200px"
        style={
          Object {
            "marginBottom": "-50px",
            "marginRight": "-150px",
            "transform": "rotate(-225deg)",
          }
        } />
      <SolidLeaf
        className="locked-bottom locked-right"
        color="text-tertiary"
        size="130px"
        style={
          Object {
            "marginBottom": "-100px",
            "marginRight": "-10px",
            "transform": "rotate(-270deg)",
          }
        } />
      <StripedLeaf
        className="locked-bottom locked-right"
        color="text-tertiary"
        size="90px"
        style={
          Object {
            "marginBottom": "60px",
            "marginRight": "-20px",
            "transform": "rotate(180deg)",
          }
        } />
      <SolidLeaf
        className="locked-bottom locked-left"
        color="text-secondary"
        size="250px"
        style={
          Object {
            "marginBottom": "-90px",
            "marginLeft": "-150px",
            "transform": "rotate(-135deg)",
          }
        } />
      <StripedLeaf
        className="locked-bottom locked-left"
        color="text-tertiary"
        size="120px"
        style={
          Object {
            "marginBottom": "-20px",
            "marginLeft": "0px",
            "transform": "rotate(-90deg)",
          }
        } />
    </Leaves>
  </div>
  <div
    className="soft-double-ends push-double-ends soft-sides text-center">
    <div
      className="grid"
      style={
        Object {
          "verticalAlign": "middle",
        }
      }>
      <Image
        url="//s3.amazonaws.com/ns.assets/apollos/annual+report/KhaleeBeard.jpg" />
      <Body>
        <blockquote>
          <p>
            I have a lot more joy in my life now.
          </p>
        </blockquote>
        <p>
          \"People thought I had been saved and I knew Jesus, but the reality was that I didnâ€™t. At Gauntlet X, I realized I knew Him in my head but not in my heart. I prayed to receive Christ and was baptized in the ocean. I am really a different person. I was really insecure all throughout high school, but I have a lot more joy in my life now.\"
        </p>
        <h6
          className="italic text-dark-primary"
          style={
            Object {
              "fontFamily": "ff-meta-serif-web-pro, serif",
            }
          }>
          Khalee Beard, Columbia Campus
        </h6>
      </Body>
    </div>
  </div>
  <div
    className="hard-sides soft-ends background--light-secondary soft-double-ends@lap-wide-and-up">
    <h3
      className="text-center soft-half-top flush-bottom">
      Gauntlet X:
    </h3>
    <h5
      className="text-center soft-half-bottom flush-bottom italic"
      style={
        Object {
          "fontFamily": "ff-meta-serif-web-pro, serif",
        }
      }>
      Every Story Matters To God
    </h5>
    <Connect(List)
      items={
        Array [
          Object {
            "count": "4,539",
            "label": "Students",
          },
          Object {
            "count": "1,066",
            "label": "Salvations",
          },
          Object {
            "count": "1,544",
            "label": "Baptisms",
          },
          Object {
            "count": "1,324",
            "label": "Volunteers",
          },
        ]
      } />
  </div>
</div>
`;
